radixdlt/src/main/java/org/radix/api/http/RadixHttpServer.java

```diff
@@ -282,6 +289,8 @@ private void addRestRoutesTo(RoutingHandler handler) {
 			respond(result, exchange);
 		}, handler);
 
+		addRoute("/api/bft", Methods.PUT_STRING, this::handleBftState, handler);
```

Comment:
What's the use case for this given we have a JSON-RPC for this already?

Reply:
To make it easier to make this inaccessible using nginx.  For the JSON RPC will need to statefully inspect the stream, which we probably don't want to do in a real system.